src/notifications/contracts/model-schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "to": "String",
+  "type": "String",
+  "title": "String",
+  "message": "String",
+  "unread": {
+    "type": "Boolean",
+    "default": true
+  },
+  "sent": {
+    "type": "Boolean",
+    "default": false
+  },
```

Comment:
Can't agree more, if you can @EddSuarez create a ticket for that with the details and the explanation of why you want ``readAt``.